src/PostgREST/Types.hs

```diff
@@ -183,8 +183,14 @@ specifiedProcArgs keys proc =
 procReturnsScalar :: ProcDescription -> Bool
 procReturnsScalar proc = case proc of
   ProcDescription{pdReturnType = (Single (Scalar _))} -> True
+  ProcDescription{pdReturnType = (SetOf  (Scalar _))} -> True
```

Comment:
Now my Haskell expertise comes to an end.. how can I make those two lines one? ie pattern match on both constructors at once, while keeping the `(Scalar _)`?

Reply:
Oh, that's a GHC limitation. Or patterns are not supported. There was a [proposal](https://github.com/osa1/ghc-proposals/blob/or_patterns/proposals/0000-or-patterns.rst#11motivation) some time ago but it didn't get finished. These cases usually need workarounds. Such as guards, multiway if or adding an extra function.